test/bun.js/our-assert.js

```diff
@@ -0,0 +1,145 @@
+import { expect } from "bun:test";
```

Comment:
instead of `our-assert`, maybe `node-test-helpers`?

Reply:
Yeah that makes sense